test/functorch/test_ac.py

```diff
@@ -1,11 +1,22 @@
 # Owner(s): ["oncall: pt2"]
 import random
+import unittest
+from math import prod
 
 import torch
 import torch._functorch.config as config
 from torch.testing._internal.common_utils import run_tests, TEST_WITH_ROCM, TestCase
 from torch.testing._internal.inductor_utils import HAS_CUDA
-from torch.utils.flop_counter import FlopCounterMode
+from torch.utils._triton import has_triton
+from torch.utils.flop_counter import FlopCounterMode, register_flop_formula
+
+
+if has_triton():
+    # note: if we only import triton in the test, the test fails:
+    # def relu_kernel_(inp_ptr, out_ptr, sz, BLOCK_SIZE: tl.constexpr):
+    # NameError('tl is not defined')
+    import triton
+    import triton.language as tl
```

Comment:
I'm confused, this should work if you import it in the test?

Reply:
filed an issue here: https://github.com/pytorch/pytorch/issues/132534